### PR TITLE
feat(membership-form): added a clear join button on the `/join` page

### PR DESCRIPTION
**Pull request by bliutech**
On `/join`, many members expressed confusion on where exactly the sign up form was to join the club. This feature adds a large button to try and make it clear where the form is.

### CODE REVIEW

File: styles/Join.module.scss [hunk 1]
```diff
@@ -3,3 +3,20 @@
   margin: 2rem auto;
   display: block;
 }
+
+button.membershipForm {
+  display: flex;
+  margin: 2rem auto 0;
+  padding: 1rem 2rem;
+  background-color: var(--cyber-dark);
```
Comment (Arc-blroth) [inline styles/Join.module.scss:11]: this color does not actually exist

File: styles/Join.module.scss [hunk 2]
```diff
@@ -3,3 +3,20 @@
...
+  border: 3px var(--cyber-gold) solid;
+  border-radius: 1rem;
+  color: var(--cyber-gold);
+  font-family: "Poppins";
+  font-size: larger;
+}
+
+button.membershipForm:hover {
+  background-color: var(--cyber-gold);
+  color: var(--cyber-dark);
```
Comment (Arc-blroth) [inline styles/Join.module.scss:21]: idk if this is a firefox bug but can we add `cursor: pointer;` because that doesn't seem to have applied by default

**Review by Arc-blroth — CHANGES_REQUESTED** [review]

File: styles/Join.module.scss (hunk 1 above)
Comment (bliutech) [reply styles/Join.module.scss:11]: Oops. 🤡

**Review by Arc-blroth — APPROVED** [review]
appears satisfactory from my perspective

**Pull request merged**